tests/phpunit/data/translation.po

```diff
@@ -51,4 +51,4 @@ msgstr "аз & ти не сме &amp;"
 #. Me, myself & Irene
 msgctxt "with_comment"
 msgid "baba"
-msgstr "баба"
\ No newline at end of file
+msgstr "баба"
```

Comment:
You need the final return on this line otherwise the output won't match (see failing test).